frontend/src/providers/GemProvider.vue

```diff
@@ -25,7 +22,7 @@ const protoGemToGemMapper = (protoGem: ProtoGem): Gem => {
       lat: protoGem.getLatitude(),
       lng: protoGem.getLongitude(),
     },
-    createdAt: dayjs(protoGem.getCreatedAt().toDate()),
+    createdAt: dayjs(protoGem.getCreatedAt()?.toDate()),
```

Comment:
I think this type mismatch was already the case, just that it didn't surface until I added `@types/google-protobuf` so that `frontend/src/api/interceptors/auth.ts` wouldn't be untyped.

Reply:
i don't think there is. Protobuf really emphasises backward compatibility, so all fields have to be optional iirc